repy.py

```diff
@@ -110,21 +106,6 @@ def get_safe_context(args):
   # Allow some introspection by providing a reference to the context
   usercontext["_context"] = usercontext
 
-  # BAD:REMOVE all API imports
-  usercontext["getresources"] = nonportable.get_resources
```

Comment:
This fix works! See [comment](https://github.com/SeattleTestbed/repy_v2/pull/127/files#r74619109) below.